packages/dd-trace/test/config.spec.js

```diff
@@ -359,6 +359,11 @@ describe('Config', () => {
     expect(config).to.have.nested.deep.property('tracePropagationStyle.extract', ['datadog', 'tracecontext', 'baggage'])
     expect(config).to.have.nested.deep.property('tracePropagationStyle.inject', ['datadog', 'tracecontext', 'baggage'])
     expect(config).to.have.property('tracing', true)
+    expect(config).to.have.nested.property('experimental.aiguard.enabled', false)
+    expect(config).to.have.nested.property('experimental.aiguard.endpoint', undefined)
+    expect(config).to.have.nested.property('experimental.aiguard.maxMessagesLength', 16)
+    expect(config).to.have.nested.property('experimental.aiguard.maxContentSize', 512 * 1024)
+    expect(config).to.have.nested.property('experimental.aiguard.timeout', 10_000)
```

Comment:
can we have those sorted alphabetically with the rest of the assertions ? basically move them a bit higher next to the other experimental stuff

Reply:
Changed, thanks!